<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeobaseLoadFiles</class>
 <widget class="QDialog" name="GeobaseLoadFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка чертежей</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Каталог</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDirectory">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tbFiles">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <column>
      <property name="text">
       <string>Имя файла</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Исх. версия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Исх. размер</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Имя в Базе</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Тип</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Комментарии</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Версия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Размер</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID сущ.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hash</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Edge num</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCheckBox" name="cbMakeFilename">
       <property name="text">
        <string>Сформировать имя в Базе</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Разделитель частей</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbSplitter">
            <item>
             <property name="text">
              <string>(нет)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>- (минус)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>_ (подстрочник)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Замена плохих символов</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbChanger">
            <item>
             <property name="text">
              <string>- (минус)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>_ (подстрочник)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbUseFN">
            <property name="text">
             <string>Использовать имя файла если тип не определён</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1,1,1,4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Префикс</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lePrefix"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbFNPart1">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <item>
               <property name="text">
                <string>Номер заказа</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Номер участка</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Тип</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Тип+номер</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Исх. имя файла</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbFNPart2">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <item>
               <property name="text">
                <string>(нет)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Номер заказа</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Номер участка</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Тип</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Тип+номер</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Исх. имя файла</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbFNPart3">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <item>
               <property name="text">
                <string>(нет)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Номер заказа</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Номер участка</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Тип</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Тип+номер</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Исх. имя файла</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Постфикс</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lePostfix"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Обработать в Автокаде</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbLoadToBase">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Загрузить в Базу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDelete_from_list">
   <property name="text">
    <string>Удалить из списка</string>
   </property>
  </action>
  <zorder>tbFiles</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeobaseLoadFiles</receiver>
   <slot>Accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeobaseLoadFiles</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Accept()</slot>
 </slots>
</ui>
